### ISSUE TITLE
Finetuning requires Pose Network Weights

### ISSUE THREAD

**Issue by sg130** [JiawangBian/Unsupervised-Indoor-Depth#9]
Thanks for sharing your work!
I am attempting to finetune the model on my custom indoor video of around 400 paired frames. I have ensured there is neither too much motion nor no motion at all. The subsequent images have almost translation-only motion. However, my fine-tuning doesn't converge.

I guess it might be due to the mismatch between Pose & Depth Networks. The link shared in this repository contains only the Depth Network trained on NYUv2. Thus, I have been using Pose Network trained on KITTI from SC-SfMLearner-Release repo, with this Depth Network.

Could you kindly upload the following models, so that we can attempt fine-tuning on custom datasets and RBGD 7Scenes?

1. Depth + Pose Network trained on Rectified NYUv2 (starting from ImageNet pretrained)
2. Depth + Pose Network trained on Original NYUv2 (without rectification) (starting from ImageNet pretrained)

Additionally, could you also share some pointers on ensuring convergence such as what masks or loss weights to use; how many epochs to finetune; finetune only part of the network? Thank you for your help

**Comment by JiawangBian** [comment]
https://1drv.ms/u/s!AiV6XqkxJHE2mQQJ30ojI5Zp5Is3?e=9Av3ML

I often use 256x320 resolution and batchsize=8.

**Comment by sg130** [comment]
Thank you very much for sharing the model weights. I shall conduct a couple more experiments and get back here if I still face any non-convergence issue. Thank you!
